diesel_tests/tests/expressions/ops.rs

```diff
@@ -199,3 +199,75 @@ fn precedence_with_parens_is_maintained() {
     let x = select((2.into_sql::<Integer>() + 3) * 4).get_result::<i32>(&connection());
     assert_eq!(Ok(20), x);
 }
+
+#[test]
+#[cfg(not(feature = "postgres"))]
```

Comment:
That should only be tested for mysql. Sqlite has no support for unsigned integers. (Or better, it accepts that you write `unsigned integer`, but will use just a normal integer instead).
```suggestion
#[cfg(feature = "mysql")]
```

Reply:
Although sqlite does not rely on the Unsigned<T> type, I suggest that we keep these tests turned on for sqlite as the tests do verify that sqlite unsigned columns continue to work as you described above (currently not covered by any tests)

Reply:
Just to clarify that: Sqlite does handle `UNSIGNED INTEGER` just the same way as `INTEGER` (or every other type that contains `INT`). Also diesel does infer these types as `Integer` not as `Unsigned<Integer>`. That means for sqlite this test would be just a duplicate of the existing tests above. That would be confusing, because the tests would be named `unsigned_*` but test something differently.

Reply:
Done.  I removed the sqlite unsigned migration and set feature = mysql on all of the tests.